asset/c.css

```diff
@@ -248,6 +258,49 @@ body.easter .note
   font-family: Arial;
 }
 
+.stats-list .stats-list-row {
+  display: block;
+}
+.stats-list .stats-list-row:nth-child(n+6) {
+  display: none;
+}
+
+.show-list {
+  overflow-y: scroll;
```

Comment:
I wonder how it looks like with a scrollbar :smile:

Reply:
This is enabled only on specific click